### PR TITLE
Add  transit gateway type

### PR DESCRIPTION
**Pull request by odg0318**
I added transit gateway type.

Related to https://github.com/k1LoW/awspec/issues/470

### CODE REVIEW

**Comment by k1LoW** [comment]
Hi @odg0318 !!! GREAT !!! Thank you !!

**Comment by k1LoW** [comment]
Released as v1.20.0.

File: doc/_resource_types/transit_gateway.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+### exist
+
+```ruby
+describe tgw('tgw-1234567890abcdefg') do
```
Comment (odg0318) [inline doc/_resource_types/transit_gateway.md:4]: @k1LoW I found my mistake here.
```ruby
describe transit_gateway('tgw-1234567890abcdefg') do
```

Comment (k1LoW) [reply doc/_resource_types/transit_gateway.md:4]: Oh! I'll fix it !! Thanks !

Comment (k1LoW) [reply doc/_resource_types/transit_gateway.md:4]: Fixed !

**Comment by k1LoW** [comment]
Released as v1.21.0.

**Pull request merged**